docs/team.rst

```diff
@@ -0,0 +1,128 @@
+Read the Docs Team
+==================
+
+readthedocs.org is the largest open source documentation hosting service.
+It's provided as a free service to the open source community,
+and is worked on by a community of volunteers that we're hoping to expand!
+We currently serve over 20,000,000 pageviews a month,
+and we hope to maintain a reliable and stable hosting platform for years to come.
+
+There are three major parts of this work:
+
+* Support
+* Operations
+* Development
+
+I'll talk about each below,
+as well as mentioning who is on the team.
+
+.. note:: You may notice that a number of names appear on multiple teams.
+          This is because we are lacking contributors.
+          So please be bold and go ahead and contact us,
+          and we'll get you sorted into the right team.
+
+Support Team
+------------
+
+Read the Docs has thousands of users who depend on it everyday.
+Every day at least one of them has an issue that needs to be addressed by a site admin.
+This might include tasks like:
+
+* Resetting a password
+* Asking for a project name to be released
+* Various other support issues
```

Comment:
"This might include tasks like ... various other support issues" doesn't mean much :) Another basic example here would be better -- ie: "Troubleshooting build errors"